debugprofiler.md

```diff
@@ -0,0 +1,52 @@
+Forge Debug Profiler oraz jego interpretacja
+===============================================
+
+Spis treści
+-----------
+* [Podstawy](#podstawy)
+* [Jak używać debugera](#Jak_używać_debugera)
+* [Czytanie Profilera](#czytanie)
+
+## Podstawy
+Minecraft dostarcza nam `Debug Profiler` dzięki któremu można sprawdzić obciążający kod.
+Szczególnie chodzi tutaj o rzeczy pokroju `TickEvent` lub `Ticking TileEntities`, 
+przydaje się to w szcególności modderom, jak także właścicielom serwerów, którzy chcą sprawdzić przyczynę lagów.
+
+<a name="Jak_używać_debugera"><h2>Jak używać profilera</h2></a>
+
+Profiler jest banalnie prosty w obsłudze:
+- `/debug start` włącza nam Profiler (z konsoli, lub bezpośrednio w grze*)
+- `/debug stop` zatrzymuje nam Proflier (z konsoli, lub bezpośrednio w grze*)
+
+**Czym dłużej Proflier będzie włączony tym dokładniejsze dane ukaże. Zaleca się conajmniej minutę.**
+
+Ważne jest by podczas profilowania robić to co zawsze, chodzić po świecie etc. Można też przteleportować się w miejsce gdzie występują lagi. **Profiler nie zbiera danych które nie istnieją, tj. `Entity` jak i `TileEntities`, dlatego ważne jest by załadować chunki na których występują lagi**.
```

Comment:
Literówki, interpunkcja.

Reply:
Niestety nie jestem w stanie ich zauważyć, jeśli mozesz to je wypisz

Reply:
21: `Proflier`, brak przecinku przed `tym`,
23: `przteleportować`, brak przecinku przed `gdzie` oraz `na których`.